src/jdk.jfr/share/classes/jdk/jfr/internal/query/view.ini

```diff
@@ -400,6 +400,28 @@ table = "COLUMN 'Method', 'Samples', 'Percent'
          SELECT stackTrace.topFrame AS T, COUNT(*), COUNT(*)
          FROM ExecutionSample GROUP BY T LIMIT 25"
 
+[application.cpu-time-hot-methods]
+label = "Java Methods that Executes the Most from CPU time sampler"
+table = "COLUMN 'Method', 'Samples', 'Percent'
+         FORMAT none, none, normalized
+         SELECT stackTrace.topFrame AS T, COUNT(*), COUNT(*)
+         FROM CPUTimeExecutionSample GROUP BY T LIMIT 25"
+
+[application.cpu-time-failed-samples]
+label = "Failed CPU Time Samples"
+table = "COLUMN 'Failed?', 'Samples', 'Percent'
+         FORMAT none, none, normalized
+         SELECT stackTrace.isNull AS T, COUNT(*), COUNT(*)
+         FROM CPUTimeExecutionSample
+         GROUP BY T"
+
+[application.cpu-time-overflowed-samples]
+label = "CPU Time Samples not recorded because of internal throttling"
```

Comment:
Align rename for CPUTimeSampleLoss

Reply:
Why are events with stack trace = null emitted? Isn't CPUTimeExecutionSampleLoss sufficient?

Reply:
This happens when the JVM is unable to walk a specific stack trace, but the queues are not full, and when the JVM is unable to resolve `Method` pointers.

The main idea behind it is that we still record that the thread runs on the CPU at this point in time; we just don't have any proper information on the stack trace. Only knowing that a thread run is still valuable information.

Reply:
Indeed. This is important for getting the correct distribution. Since we are sampling per thread cpu time, having one global 'event loss' is suboptimal as it is not possible to easily assign the lost events to threads proportionally.
But because keeping the event loss per thread would be too expensive we settled for this solution. OTOH, for unwalkable samples we can 'cheaply' record them and maintain the distribution.

Reply:
That's a good idea. So a failed sample would have `failed=true,stackTrace=null`?